runtime/src/tasks/qthreads/tasks-qthreads.c

```diff
@@ -555,15 +556,15 @@ static void setupCallStacks(void) {
     size_t envStackSize;
     char newenv_stack[QT_ENV_S];
 
-    chpl_bool guardPagesEnabled;
+    size_t guardPagesEnabled;
```

Comment:
Maybe `guardPageSize`?  That is what it holds, right?  And zero if we're not using them.

Reply:
Oh.  No, it's still a boolean, it's just that now it's a C boolean residing in a `size_t` container so that we can easily do arithmetic on it.  The `size_t` still seems misleading to me.  Maybe just plain `int`?